posthog/temporal/workflows/__init__.py

```diff
@@ -65,4 +67,5 @@
     squash_events_partition,
     update_batch_export_backfill_model_status,
     update_export_run_status,
+    cohort_for_flag,
```

Comment:
Let's not mix these two. I think we need a separate list of activities per team/service.

Reply:
For example, for data warehouse, we just rolled out the `data-warehouse-task-queue` Temporal task queue, and a set of workers to service that queue. We should do the same here: Have a separate set of activities going to a new task queue. We can then copy the release made for data warehouse and deploy workers.

Reply:
One reason for this is that, at a glance, the code appears to be mostly sync Python. This will block batch exports which run with the default event loop executor (and use mostly async Python). This activity would perhaps be better serviced by a `ThreadPoolExecutor` (or by migrating to async code, but afaik our version of django doesn't support that).

Reply:
I can update this PR to match after this one is merged https://github.com/PostHog/posthog/pull/18700